Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -294,7 +295,7 @@ public class ArcGISARView: UIView {
             locationManager.requestAlwaysAuthorization()
         }
         else{
-            didStartOrFailWithError(ArcGISARView.clError.missingPListKey)
+            didStartOrFailWithError(ArcGISARView.CoreLocationError.missingPListKey)
```

Comment:
within ArcGISARView you don't need to qualify `ArcGISARView.CoreLocationError.missingPListKey`. You can just write `CoreLocationError.missingPListKey`

Reply:
Yes, thank you.